### ISSUE TITLE
Overwriting DEBUG setting

### ISSUE THREAD

**Issue by yES** [django-nose/django-nose#160]
For what purposes is written that line of code?
`settings.TEMPLATE_DEBUG = settings.DEBUG = False`

https://github.com/django-nose/django-nose/blob/master/django_nose/testcases.py#L119

It overwrite using of decorator for test class. For example:

```
from django_nose import FastFixtureTestCase
from django.test.utils import override_settings

@override_settings(DEBUG=True)
class SomeTestClass(FastFixtureTestCase):

    def setUp(self):
         if not settings.DEBUG:
             print 'My settings.DEBUG is False, but I want to set it to True'
```

And I think it's bad practice.

**Comment by hwkns** [comment]
FWIW, the `FastFixtureTestCase` class will be obsolete in Django 1.8:
[Django 1.8 TestCase data setup](https://docs.djangoproject.com/en/1.8/releases/1.8/#testcase-data-setup)

**Comment by karabijavad** [comment]
actually, this is something i just encountered too...
why is debug being set to false for this test case?

**Comment by hwkns** [comment]
There is a comment about it:
```
    def _pre_setup(self):
        """Disable transaction methods, and clear some globals."""
        # Repeat stuff from TransactionTestCase, because I'm not calling its
        # _pre_setup, because that would load fixtures again.
        cache.cache.clear()
        settings.TEMPLATE_DEBUG = settings.DEBUG = False
```
But I just looked at the Django source, several versions back, and didn't find anything like that in `TransactionTestCase._pre_setup()`.  It looks like @erikrose copied that part of the code from `django-test-utils`, so it probably originated here: https://github.com/ericholscher/django-test-utils/blame/fc2c6194cb602e2d17e43814eb65cc6bf9f5898c/test_utils/test_runners/keep_database.py#L41  I agree, it's bad practice.

**Comment by karabijavad** [comment]
weird.

i guess its because, when running your unit tests, you should not be connected to any production services anyway (like stripe, or postgres), so you might as well run with debug off, for the unit tests to reflect production?

kinda makes sense i guess...

i like debug on though, so i can see more logging etc in my output

**Comment by jwhitlock** [comment]
I agree, this and the cache clearing seem wrong. @erikrose - any reason to keep it?  Does someone want to submit a PR?
